fusor-ember-cli/app/routes/deployment.js

```diff
@@ -149,7 +149,13 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
     error: function(reason) {
       console.log(reason);
       var controller = this.controllerFor('deployment');
-      controller.set('errorMsg', reason.responseJSON.error.message);
+      var msg = null;
+      if (reason.responseJSON) {
+        msg = reason.responseJSON.error.message;
+      } else if (reason.message) {
+        msg = reason.message;
```

Comment:
@knowncitizen, I removed the generate error message. `reason.message` should show the JS errors rather than looking in the console. Not sure we want to merge this, but it could help us with debugging if QA or other's send us the error message.